layouts/partials/docs/article.html

```diff
@@ -8,7 +8,7 @@
         {{ partial "nav.html" . }}
       </div>
 
-      <div class="column">
+      <div class="column is-three-quarters">
```

Comment:
Don't we want this to be 3/4s only when the left nav is showing?

Reply:
The column system uses Flexbox to automatically expand and contract these.

Reply:
I can confirm that this change leads to the problem I mentioned at the start of this thread, see #128 

I'll be revert this specific change, reopen the issue and submit a fix.